<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI</class>
 <widget class="QWidget" name="UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>UI</string>
  </property>
  <widget class="QLabel" name="WLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY헤드라인M</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>오늘의 날씨</string>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY헤드라인M</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>날씨 그림</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>380</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY헤드라인M</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>음성인식</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>308</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item>
     <widget class="QPushButton" name="MirrorButton1">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>150</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>거울모드</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SettingButton">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>150</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/setting.png</normaloff>
        <normalon>:/images/setting.png</normalon>images/setting.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="Fan">
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>선풍기</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="FanButton">
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Off</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="AirLabel">
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>에어컨</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="AirButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>On</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>390</y>
     <width>231</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QPushButton" name="SchButton">
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>일정</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="SchLabel">
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>일정 내용</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>240</y>
     <width>228</width>
     <height>109</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QPushButton" name="MindButton">
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>감정인식</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>기능2 버튼</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>510</y>
     <width>281</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="TimeLayout"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>231</width>
     <height>74</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="ULtemperature">
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>온도</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ULhumidity">
      <property name="font">
       <font>
        <family>HY헤드라인M</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>humi</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
